node/src/neighborhood/mod.rs

```diff
@@ -800,12 +804,28 @@ impl Neighborhood {
             payload_size: 10000,
         };
         if self.handle_route_query_message(msg).is_some() {
-            self.overall_connection_status.update_can_make_routes(true);
+            debug!(
+                &self.logger,
+                "The check_connectedness() found a 3 hops route for the first time."
+            );
+            self.overall_connection_status
+                .update_ocs_stage_and_send_message_to_ui(
+                    OverallConnectionStage::ThreeHopsRouteFound,
+                    self.node_to_ui_recipient_opt
+                        .as_ref()
+                        .expect("UI was not bound."),
+                    &self.logger,
+                );
             self.connected_signal_opt
                 .as_ref()
                 .expect("Accountant was not bound")
                 .try_send(StartMessage {})
                 .expect("Accountant is dead")
+        } else {
+            debug!(
+                &self.logger,
+                "The check_connectedness() still can't found a good route."
```

Comment:
"found" is the wrong tense. Probably you want "find." Also, maybe find another way to say "The check_connectedness()."